src/pages/permissions/panels/EditGroupPermissionsForm.tsx

```diff
@@ -38,6 +40,12 @@ const EditGroupPermissionsForm: FC<Props> = ({
 }) => {
   const notify = useNotify();
   const [search, setSearch] = useState("");
+  const { canViewPermissions } = useServerEntitlements();
+  const { canEditGroup } = useGroupEntitlements();
+  const permissionEditRestriction =
+    !canViewPermissions() || !canEditGroup(group)
+      ? "You do not have permission to edit entitlements for this group"
+      : "";
```

Comment:
Shall we cater the message better towards the combination of the two permissions?
I think we should say something different if you can edit the group, but not view permissions, to help users fix their permissions with the right information.